README.md

```diff
@@ -69,6 +69,11 @@ from the the project root, issue the following command:
 
     pip install -Ur requirements.txt
 
+###Create test users
+Some of the tests require logging in as an user, and others require logging in as an editor user or paypal user.
+To run these tests you will need to create accounts on https://addons.allizom.org/. If you want to run the tests
+that require editor access, then you will need to ask someone on IRC to upgrade one of your test accounts.
```

Comment:
There's no need for an 'editor' user. The 'editable' user is a user reserved for editing the user profile without impacting tests that rely on the profile to be static. We're actually working on switching to Firefox Accounts for AMO, which will mean there's going to be no need for `variables.json` to include an AMO user. That said, we can still take this improvement to the README in the short term, but both 'default' and 'editable' are basic accounts.